### ISSUE TITLE
Unable to publish a news into created targets

### ISSUE THREAD

**Issue by boubaker** [Meeds-io/meeds#2281]
**Is your feature request related to a problem? Please describe.**
**Scenario**:
1/ Create a user
2/ Add user into "`Web contributors`" as *
3/ Add user into "`Administrators`" as *
4/ Add NewsList into a page by creating a dedicated target with permission to "`Users`"
5/ Try to write an article in a dedicated space and publish it into created Target

**Result**
* Can't publish the news into newly created Target
* Can't publish even when adding user as publisher in group `Web contributors`
* The message says
> No target are allowed for you. Please contact your administrators.
While I answer :
> I am using a user who is an administrator and who gives rights to users, tell me more about what permissions are remaining. (Of course no AI so no interaction :) )

**Describe the solution you'd like**
Give access to `Web Contributors` to publish to all targets outside of spaces
+

**Describe alternatives you've considered**
Give clear steps to the administrator (instead of a generic message) to know how to publish to targets where he's not allowed to. Thus we should replace the "Contact...." by the list of available targets + disabled effect on not permitted targets with an icon + tooltip on icon = "Not publisher of Users/XYZ group"

**Comment by srenault-meeds** [comment]
cc @Julien-Dubois-eXo have you catched this one?

**Comment by Julien-Dubois-eXo** [comment]
Hello no as you have added User as permission all users with publish permission on the space should be able to use the target. It's a bug. 

However, it is expected not to have the solution you would like. More than ever with the automatic creation of target at the space creation. If we do that the web contributors will have an endless list of targets available.  

To avoid the bad UX of the message, we will simply not display the publish option if a user has no targets allowed. To publish your should have both conditions : 

1. Permission to publish = space publisher or super publisher (web contributor)
2. Targets allowed for you = permission for a space you are member of or permission of a group you are member of

**Comment by srenault-meeds** [comment]
>  If we do that the web contributors will have an endless list of targets available.

Not a problem I think.
As an admin, I'd rather have access to all targets so I can manage my publishing.
If you don't want to get the whole list, then don't be an admin IMHO

**Comment by Julien-Dubois-eXo** [comment]
> Not a problem I think.


I'm afraid I have to disagree we have several customer cases where it will be a problem. @margondicco is against too.

**Comment by srenault-meeds** [comment]
Up?

**Comment by Julien-Dubois-eXo** [comment]
As discussed yesterday, it's no longer prioritized.

**Comment by srenault-meeds** [comment]
I am not sure to understand because we have talked about it. Indeed, not prioritized (this is why this normal priority I guess) but if possible to provide it as enhancement during the TC. 
BTW, I have the feeling this is duplicated with https://github.com/Meeds-io/meeds/issues/2855

Am I wrong?

**Comment by Julien-Dubois-eXo** [comment]
Yes for both. I was in a meeting, I close the other I created recently.
